### ISSUE TITLE
No alert for "succeeded" pods

### ISSUE THREAD

**Issue by Napsty** [Napsty/check_rancher2#1]
When a workload is updated or redeployed, the currently running pods of that workload will change from "running" to "succeeded" when the new pods were started. With the current version of check_rancher2, this will result in a CRITICAL alert:

```
Additional Info: CHECK_RANCHER2 CRITICAL - Pod "internal-api-cron-test-1541435700-2jqh5" is succeeded - Pod "internal-api-cron-test-1541435760-h5hrh" is succeeded - Pod "internal-api-cron-test-1541435820-t58bs" is succeeded - Pod "internal-api-cron-test-1541435880-5ptfv" is succeeded -
```

This should result in a WARNING, not CRITICAL.

**Comment by Napsty** [comment]
Actually "succeeded" is a correct (healthy) state for job workloads, e.g. a cronjob:

![image](https://user-images.githubusercontent.com/3702203/48250549-fb2c8f80-e3fe-11e8-9456-f33ff77d4b49.png)

See https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/

This basically means that a pod is started up to run a command and is then shut down again. If the command was running successfully, the state will be "succeeded". 

In kubectl we can verify this, too. However the status is shown as "completed":

```
$ kubectl get pod --namespace gamma -a | grep cron
internal-api-cron-newsletter-1541743200-9n24p           0/1       Completed   0          2h
internal-api-cron-noindex-1541739600-vbpqv              0/1       Completed   0          3h
internal-api-cron-notifications-1541739600-zwth9        0/1       Completed   0          3h
internal-api-cron-regio-report-daily-1541746800-wnlqv   0/1       Completed   0          1h
```

According to https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/:

> Succeeded = All Containers in the Pod have terminated in success, and will not be restarted.

TL;DR: Succeeded should not give a WARNING nor a CRITICAL as it clearly is a healthy state. It is up to the user to correctly deploy job workloads.

**Comment by Napsty** [comment]
Fixed in https://github.com/Napsty/check_rancher2/commit/52b7dde04621d9af7ed24f023f34c349f740e3e6
